### PR TITLE
Create information email for volunteers

### PR DESCRIPTION
**Pull request by anton-firmware**

### CODE REVIEW

**Review by Adimote — DISMISSED** [review]
There's no content, please only add reviewers when it's ready to review. Arguably you shouldn't even create a PR until it's ready

**Comment by RealOrangeOne** [comment]
*Highly* agree with not opening PRs until there's something ready to review, and has relevant commits.

Also, this repo is for communication with teams, not teams. I suggest drafting these somewhere like drive, as it's more relevant, as we don't _necessarily_ want information like this released publicly, or before it's released to teams etc.

File: SB2018-volunteer/2018-04-13.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+to: All Volunteers
+subject: Volunteer Information - SourceBots 2018
+---
+
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition! This event could not run without you and we appreciate the hardwork and effort that you will put in!
+
+Even though it is hard work, we can assure you that it will be a fun weekend filled with robot madness and making sure everything is running smoothly!
+
+To that end, we thought now would be a good time to send an email detailing information about the competition! :)
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: Unnecessary smiley

File: SB2018-volunteer/2018-04-13.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Date and location
+
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html).
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: We are setting up on Friday too, you have not included times

Comment (trickeydan) [reply SB2018-volunteer/2018-04-13.md]: Also, SUSU have multiple buildings and floors, please specify

File: SB2018-volunteer/2018-04-13.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+For the first day, please meet in the [Mountbatten Building](http://data.southampton.ac.uk/building/53.html) on **Level 4** at **0800**. Unfortunately this building does require card access so if you find yourself needing letting in please contact one of us. (**PLACEHOLDER FOR NUMBER TO BE REPLACED IN FINAL EMAIL**)
+
+### Pub 
+
+On the sign up form, it was written that we will be heading to the pub for a meal on the **22nd April** the Sunday. We have chosen instead to this on the **21st April** as we will be very busy on the Sunday setting down the arena and may not get out of The Cube until late. 
+
+Due to this, we will instead be heading to the pub on the **Saturday Evening**!!
+
+### Zello
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: This is under discussion as Zello doesn't have Privacy Shield

File: SB2018-volunteer/2018-04-13.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+to: All Volunteers
+subject: Volunteer Information - SourceBots 2018
+---
+
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition! This event could not run without you and we appreciate the hardwork and effort that you will put in!
+
+Even though it is hard work, we can assure you that it will be a fun weekend filled with robot madness and making sure everything is running smoothly!
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: This sentence reads badly.

File: SB2018-volunteer/2018-04-13.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+To that end, we thought now would be a good time to send an email detailing information about the competition! :)
+
+### Date and location
+
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html).
+
+### Roles
+
+We will be assinging roles to volunteers soon and will email you about this in due course!
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: *assinging* sounds like it may be some sort of vulgar music

**Review by trickeydan — DISMISSED** [review]
This does not pass the tests.

File: SB2018-volunteer/2018-04-13.md (hunk 1 above)
Comment (anton-firmware) [reply SB2018-volunteer/2018-04-13.md]: fixed in 9d736b9d45515412a9b0c036d0959422dfef7b3b

File: SB2018-volunteer/2018-04-13.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### Schedule for the first day
+
+For the first day, please meet in the [Mountbatten Building](http://data.southampton.ac.uk/building/53.html) on **Level 4** at **0800**. Unfortunately this building does require card access so if you find yourself needing letting in please contact one of us. (**PLACEHOLDER FOR NUMBER TO BE REPLACED IN FINAL EMAIL**)
+
+### Pub 
+
+On the sign up form, it was written that we will be heading to the pub for a meal on the **22nd April** the Sunday. We have chosen instead to this on the **21st April** as we will be very busy on the Sunday setting down the arena and may not get out of The Cube until late. 
+
+Due to this, we will instead be heading to the pub on the **Saturday Evening**!!
```
Comment (RealOrangeOne) [inline SB2018-volunteer/2018-04-13.md]: Please remove double exclamation marks

File: SB2018-volunteer/2018-04-13.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+### Date and location
+
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html) which is on the bottom floor of the Students' Union building.
+
+This will take up the whole day so if you're around please drop in between **0800-2300**.
+
+### Scaffolding Setup
+
+Our arena will be built out of scaffolding and this will arrive on the 19th April. We will need help on the **20th April** to help set up the arena in the cube. Anyone that is available should report to the cube where you'll be assigned tasks.
```
Comment (RealOrangeOne) [inline SB2018-volunteer/2018-04-13.md]: Why do we need to specify this in the email?

File: SB2018-volunteer/2018-04-13.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+subject: Volunteer Information - SourceBots 2018
+---
+
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition, even though it will be hard work, we can assure you that it will be a fun weekend filled with robot madness and endless fun.
+
+To that end, we thought now would be a good time to send an email detailing information about the competition!
+
+### Date and location
+
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html) which is on the bottom floor of the Students' Union building.
```
Comment (RealOrangeOne) [inline SB2018-volunteer/2018-04-13.md]: I suggest linking to our website instead. There's more information there, and it also links to the cube.

File: SB2018-volunteer/2018-04-13.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+### Pub 
+
+On the sign up form, it was written that we will be heading to the pub for a meal on the **22nd April** the Sunday. We have chosen instead to this on the **21st April** as we will be very busy on the Sunday setting down the arena and may not get out of The Cube until late. 
+
+Due to this, we will instead be heading to the pub on the **Saturday Evening**!!
+
+### Hats
+
+Who said only robots are allowed to look swanky at the competition? Bring your best hat and wear it throughout the weekend!!
```
Comment (RealOrangeOne) [inline SB2018-volunteer/2018-04-13.md]: Please remove double exclamation marks

File: SB2018-volunteer/2018-04-13.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+This will take up the whole day so if you're around please drop in between **0800-2300**.
+
+### Scaffolding Setup
+
+Our arena will be built out of scaffolding and this will arrive on the 19th April. We will need help on the **20th April** to help set up the arena in the cube. Anyone that is available should report to the cube where you'll be assigned tasks.
+
+### Roles
+
+We will be assigning roles to volunteers soon and will email you about this in due course!
```
Comment (RealOrangeOne) [inline SB2018-volunteer/2018-04-13.md]: Again, do we need exclamation marks here?

**Review by RealOrangeOne — DISMISSED** [review]
I still don't think this should be in this repo, as this is for _team communication_ only!

File: SB2018-volunteer/2018-04-13.md (hunk 8 above)
Comment (trickeydan) [reply SB2018-volunteer/2018-04-13.md]: The cube is not on the bottom floor!

File: SB2018-volunteer/2018-04-13.md [hunk 11]
```diff
@@ -3,23 +3,25 @@ to: All Volunteers
...
-To that end, we thought now would be a good time to send an email detailing information about the competition! :)
+To that end, we thought now would be a good time to send an email detailing information about the competition!
 
 ### Date and location
 
-As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html).
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html) which is on the bottom floor of the Students' Union building.
 
-### Roles
+This will take up the whole day so if you're around please drop in between **0800-2300**.
```
Comment (trickeydan) [inline SB2018-volunteer/2018-04-13.md]: I doubt it's useful if they drop in at 2259

**Review by trickeydan — DISMISSED** [review]

File: SB2018-volunteer/2018-04-13.md (hunk 11 above)
Comment (RealOrangeOne) [reply SB2018-volunteer/2018-04-13.md]: potentially a word change, saying we'll be working between these times

**Review by trickeydan — APPROVED** [review]
Whilst this isn't perfect, we need to ship this email *ASAP*

**Comment by RealOrangeOne** [comment]
Repo is for team emails. Moved to drive for further edits, if required https://docs.google.com/document/d/1v_CTF1SWbaMv89iwl34JWhyxziU0h-5n3oTKf2wRlaM/edit

**Pull request closed without merging**